app/controllers/update_checks_controller.rb

```diff
@@ -233,6 +234,10 @@ def unique
   end
 
   private
+    def geo_code
+      GeoCoder.broadcast(request.remote_ip)
```

Comment:
Maybe add a comment that the IP address is never stored anywhere, and only used to detect the country to show on the /geo earth visualisation.